### ISSUE TITLE
Using quarkus-hibernate-orm-panache:0.12.0 with Gradle does not start the server in dev mode

### ISSUE THREAD

**Issue by dubemarcantoine** [quarkusio/quarkus#1906]
Hi,
I am trying to get a project with Quarkus using Gradle working, but when I add the `quarkus-hibernate-orm-panache:0.12.0` dependency, the server won't start when I do `gradle quarkusDev`. However, when I remove the dependency, it starts normally. There does not seem to be an error in the Gradle logs.

**Without Panache dependency**
```
[madube@arch-desktop testing]$ gradle quarkusDev
> Task :compileJava
> Task :processResources NO-SOURCE
> Task :classes

> Task :quarkusDev
Starting process: 
java
-Xdebug
-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n
-Dquarkus-internal.devMode
-XX:TieredStopAtLevel=1
-Xverify:none
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-Dquarkus-internal.runner.classes=/home/madube/IdeaProjects/testing/build/classes/java/main
-Dquarkus-internal.runner.sources=/home/madube/IdeaProjects/testing/src/main/java
-Dquarkus-internal.runner.resources=/home/madube/IdeaProjects/testing/src/main/resources
-jar
/home/madube/IdeaProjects/testing/build/testing-dev.jar
/home/madube/IdeaProjects/testing/build/classes/java/main
/home/madube/IdeaProjects/testing/build/wiring-classes
/home/madube/IdeaProjects/testing/build/transformer-cache
Args: 
java
-Xdebug
-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n
-Dquarkus-internal.devMode
-XX:TieredStopAtLevel=1
-Xverify:none
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-Dquarkus-internal.runner.classes=/home/madube/IdeaProjects/testing/build/classes/java/main
-Dquarkus-internal.runner.sources=/home/madube/IdeaProjects/testing/src/main/java
-Dquarkus-internal.runner.resources=/home/madube/IdeaProjects/testing/src/main/resources
-jar
/home/madube/IdeaProjects/testing/build/testing-dev.jar
/home/madube/IdeaProjects/testing/build/classes/java/main
/home/madube/IdeaProjects/testing/build/wiring-classes
/home/madube/IdeaProjects/testing/build/transformer-cache
```

**With Panache dependency**
```
[madube@arch-desktop testing]$ gradle quarkusDev
> Task :compileJava
> Task :processResources NO-SOURCE
> Task :classes

> Task :quarkusDev
Starting process: 
java
-Xdebug
-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n
-Dquarkus-internal.devMode
-XX:TieredStopAtLevel=1
-Xverify:none
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-Dquarkus-internal.runner.classes=/home/madube/IdeaProjects/testing/build/classes/java/main
-Dquarkus-internal.runner.sources=/home/madube/IdeaProjects/testing/src/main/java
-Dquarkus-internal.runner.resources=/home/madube/IdeaProjects/testing/src/main/resources
-jar
/home/madube/IdeaProjects/testing/build/testing-dev.jar
/home/madube/IdeaProjects/testing/build/classes/java/main
/home/madube/IdeaProjects/testing/build/wiring-classes
/home/madube/IdeaProjects/testing/build/transformer-cache
Args: 
java
-Xdebug
-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n
-Dquarkus-internal.devMode
-XX:TieredStopAtLevel=1
-Xverify:none
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-Dquarkus-internal.runner.classes=/home/madube/IdeaProjects/testing/build/classes/java/main
-Dquarkus-internal.runner.sources=/home/madube/IdeaProjects/testing/src/main/java
-Dquarkus-internal.runner.resources=/home/madube/IdeaProjects/testing/src/main/resources
-jar
/home/madube/IdeaProjects/testing/build/testing-dev.jar
/home/madube/IdeaProjects/testing/build/classes/java/main
/home/madube/IdeaProjects/testing/build/wiring-classes
/home/madube/IdeaProjects/testing/build/transformer-cache

BUILD SUCCESSFUL in 3s
2 actionable tasks: 2 executed
[madube@arch-desktop testing]$ 

```

As you can see, the task stops instead of keeping the server running.

I created this repo to reproduce with `gradlew quarkusDev`. https://github.com/dubemarcantoine/quarkus-issue

Thanks for your help!!

**Comment by FroMage** [comment]
Hi,

Could you try with the newly released `0.13` please?

**Comment by dubemarcantoine** [comment]
@FroMage thanks for the reply! 

After updating, the Gradle task `quarkusDev` now starts and does not stop once the build is done. However, when I have the panache dependency, the server still does not start, although it does without it. `curl: (7) Failed to connect to localhost port 8080: Connection refused`

Just updated my example repo to have code examples.

**Comment by FroMage** [comment]
OK thanks.

**Comment by stalep** [comment]
Hi, I tested your repo just now (thanks for providing an easy way to test it!). We have a bug since Quarkus expects a driver for the datasource since panache is added as an dependency even if its not used. Because of this the runner.jar is empty (it is still created).

**Comment by FroMage** [comment]
Oh, that's the error? Then it's a duplicate of https://github.com/quarkusio/quarkus/issues/1732, no?

**Comment by stalep** [comment]
Ah, yup it is. It fails both for creating the runner.jar and dev-mode.

**Comment by stalep** [comment]
Duplicate of #1732

**Comment by FroMage** [comment]
Thanks.
